cpp/benchmarks/copying/gather_tests.cu

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
```

Comment:
I think rename this file to gather_benchmark.cpp (btw, I doubt it needs to be a .cu file, and compilation is much faster for .cpp files).